src/main/java/com/company/ExcelWriter.java

```diff
@@ -25,13 +23,30 @@ public ExcelWriter() throws FileNotFoundException {
         spreadsheet = workbook.createSheet("Student Data");
     }
 
-    public void writeListToAnExcelFile(HashMap<Integer, List<Integer>> companies) throws IOException {
-        int rowId = 0;
-        for(int x = 0; x < listOfSOmething.size(); x++){
-            Row row = spreadsheet.createRow(rowId++);
-            Cell cell = row.createCell(0);
-            cell.setCellValue(listOfSOmething.get(x));
-       }
+    public void createHeader(){
+        List<String> headers = Arrays.asList("Company", "Q1", "Q2", "Q3");
```

Comment:
malo to generyczne. Ze masz na sztywno wstawione 4 wartosci jako headery. Do przepisania